libs/@guardian/libs/src/measure/startMeasure.ts

```diff
@@ -31,16 +34,28 @@ export const startMeasure = (
 		start: performance.now(),
 	} satisfies PerformanceMeasureOptions;
 
-	const endMeasure = () => {
-		const measureName = serialise(team, name, action);
+	const measureName = serialise(team, name, action);
 
-		const { duration } = window.performance.measure(measureName, options) ??
+	const endMeasure = (end?: number) => {
+		const { duration } = window.performance.measure(measureName, {
+			...options,
+			end,
+		}) ??
 			window.performance.getEntriesByName(measureName, 'measure').at(-1) ?? {
 				duration: performance.now() - options.start,
 			};
 
 		return Math.ceil(duration);
 	};
 
-	return { endMeasure };
+	const observe = () =>
```

Comment:
I feel like the name could describe what the function does better. 

Just an idea but  `observeEndMark()`?